metricbeat/metricbeat.reference.yml

```diff
@@ -1267,3 +1267,19 @@ logging.files:
 
 # Set to true to log messages in json format.
 #logging.json: false
+
+
+#================================ HTTP Endpoint ======================================
+# Each beat can expose internal data points through a http endpoint. For security
+# reason the endpoint is disabled by default. This feature is currently experimental.
+# Stats can be access through http://localhost:5066/stats For a pretty json output,
+# add ?pretty .
+
+# Defines if http endpoint is enabled
```

Comment:
`Defines if the HTTP endpoint is enabled.`

Reply:
done